File: Intranet.Service/PessoaJuridicaService.cs
using Intranet.Domain.Entities;
using Intranet.Domain.Interfaces.Repositories;
using Intranet.Domain.Interfaces.Services;
using Intranet.Domain.Services;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Intranet.Service
{
    public class PessoaJuridicaService : ServiceBase<PessoaJuridica>, IPessoaJuridicaService
    {
        public IPessoaJuridicaRepository _repository;

        public PessoaJuridicaService(IPessoaJuridicaRepository repository)
            : base(repository)
        {
            this._repository = repository;
        }

        public PessoaJuridica GetByCNPJ(string CNPJEmpresa, int CNPJFilial, int CNPJDV)
        {
            return _repository.GetByCNPJ(CNPJEmpresa, CNPJFilial, CNPJDV);
        }

        public PessoaJuridica GetByRazaoSocial(string razaoSocial)
        {
            return _repository.GetByRazaoSocial(razaoSocial);
        }
    }
}

File: Intranet.Web/scripts/Services/PessoaJuridicaService.js
angular.module('app').service('PessoaJuridicaService', function ($http) {
    var serviceBase = 'http://localhost:50837/';
    var serviceBaseProducao = 'http://192.168.1.199:9810/Intranet.API/';

    this.GetByRazaoSocial = function (razao) {
        return $http.get(serviceBase + "api/PessoaJuridica/GetByRazaoSocial?razaoSocial="+razao).then(function (response) {
            return response
        }, function (response) {
            return alert("Erro: " + response.status);
        });
    }

    this.GetByCNPJ = function (CNPJEmpresa, CNPJFilial, CNPJDV) {
        return $http.get(serviceBase + "api/PessoaJuridica/GetByCNPJ?CNPJEmpresa=" + CNPJEmpresa + "&CNPJFilial=" + CNPJFilial + "&CNPJDV=" + CNPJDV).then(function (response) {
            return response
        }, function (response) {
            return alert("Erro: " + response.status);
        });
    }

    this.GetCdPessoaJuridica = function (CNPJEmpresa, CNPJFilial, CNPJDV) {
        return $http.get(serviceBase + "api/PessoaJuridica/GetCdPessoaJuridica?CNPJEmpresa=" + CNPJEmpresa + "&CNPJFilial=" + CNPJFilial + "&CNPJDV=" + CNPJDV).then(function (response) {
            return response
        }, function (response) {
            return alert("Erro: " + response.status);
        });
    }
});